packages/php-wasm/cli/project.json

```diff
@@ -80,7 +80,7 @@
 		"start": {
 			"executor": "@wp-playground/nx-extensions:built-script",
 			"options": {
-				"scriptPath": "dist/packages/php-wasm/cli/main.js"
+				"scriptPath": "dist/packages/php-wasm/cli/php-wasm.js"
```

Comment:
What is the purpose of this? Where does this name come from? Naively, I would have expected the name to be "cli.js" based on filename "cli.ts".